HospitalGame_Common/src/common/IPowerUpItem.java

```diff
@@ -12,4 +12,5 @@
 public interface IPowerUpItem {
     double getBuff();
     long getTimeLeftOfBuff();
+    void update(long currentTime);
```

Comment:
This is not allowed since it gives access to update the object, outside the class, and from other layers